src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.Globalization.cs

```diff
@@ -35,6 +29,16 @@ public static bool Contains(this ReadOnlySpan<char> span, ReadOnlySpan<char> val
             return IndexOf(span, value, comparisonType) >= 0;
         }
 
+        /// <summary>
+        /// Returns a value indicating whether the specified span contains <see cref="char.IsWhiteSpace(char)">any
+        /// white-space characters</see>, and returns <see langword="true"/> if found. If not found, returns
+        /// <see langword="false"/>.
+        /// </summary>
+        /// <param name="span">The source span.</param>
```

Comment:
It's better to match the docs (https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.iswhitespace) for the summary/return values:

```suggestion
        /// <summary>
        /// Indicates whether the specified span contains <see cref="char.IsWhiteSpace(char)">any
        /// white-space characters</see>.
        /// </summary>
        /// <param name="span">The source span.</param>
        /// <returns><see langword="true"/> if the span contains any whitespace characters, <see langword="false"/> otherwise.</returns>
```

(Same for the other new methods in this PR.)